### ISSUE TITLE
[BUG/] this transaction may be risky

### ISSUE THREAD

**Issue by vipulm7** [PatilShreyas/EasyUpiPayment-Android#187]
Write information about Bug/Feature Request here
Google pay gives this error - this transaction may be risky.
What is the way to solve this?

**Comment by ProgramWithSP** [comment]
Did u solved this error? Please reply

**Comment by vipulm7** [comment]
No. Most probab, this is from the RBI side that we need to get verified as a merchant to receive money.

**Comment by ProgramWithSP** [comment]
But when I used my friend merchant verified account, the problem still there
